reddit_decider/__init__.py

```diff
@@ -227,48 +228,48 @@ def _get_dynamic_config_value(
 
         res = decider_func(dynamic_config_name, ctx)
         if res is None:
-            return
+            return default
         error = res.err()
         if error:
             logger.warning(f"Encountered error in decider.get_value(): {error}")
-            return None
+            return default
         else:
             return res.val()
 
-    def get_bool(self, dynamic_config_name: str) -> Optional[bool]:
+    def get_bool(self, dynamic_config_name: str, default: bool = False) -> bool:
```

Comment:
I think `feature_name`, instead of `dynamic_config_name`, was agreed in the API doc and it's more succinct.